module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include <algorithm>
+#include <array>
+#include <utility>
+#include <vector>
```

Comment:
Do you need algorithm and utility in the header? You generally prefer to reduce header include in favour of cpp includes to avoid uncliding library you do not use in the file ,because when header is included everything in it is included as well